### PR TITLE
Convert convention from WH to HW

### PR DESCRIPTION
**Pull request by yuyu2172**
Fix subset of #189 

Changes include following list of directories.

+ transforms/image
+ transforms/bbox
+ transforms/keypoint
+ links/models/faster_rcnn
+ links/model/ssd
+ links/model/segnet
+ utils/testing/generate_random_bbox

### CODE REVIEW

File: chainercv/transforms/image/ten_crop.py [hunk 1]
```diff
@@ -25,13 +25,13 @@ def ten_crop(img, size):
...
 
     Returns:
         The cropped arrays. The shape of tensor is :math:`(10, C, H, W)`.
 
     """
-    W, H = size
+    H, W = size
     iH, iW = img.shape[1:3]
 
     if iW < W or iH < H:
```
Comment (Hakuyume) [inline chainercv/transforms/image/ten_crop.py]: `if iH < H or iW < W:`

File: chainercv/transforms/image/resize_contain.py [hunk 2]
```diff
@@ -72,23 +72,24 @@ def _get_pad_slice(img, size):
     """
```
Comment (Hakuyume) [inline chainercv/transforms/image/resize_contain.py]: `(max_H, max_W)`?

File: chainercv/transforms/image/scale.py [hunk 3]
```diff
@@ -34,12 +34,12 @@ def scale(img, size, fit_short=True):
 
     if fit_short:
         if W < H:
```
Comment (Hakuyume) [inline chainercv/transforms/image/scale.py]: I prefer `if H < W:`.

**Comment by Hakuyume** [comment]
Could you fix `resize_keypoint`?

**Comment by yuyu2172** [comment]
Updated.

**Comment by yuyu2172** [comment]
PR to different branch.

**Pull request closed without merging**